src/studio/src/designer/backend.Tests/Controllers/DataModelsController/PutDatamodelTests.cs

```diff
@@ -52,7 +52,7 @@ public class PutDatamodelTests : ApiTestsBase<DatamodelsController, PutDatamodel
 
     public PutDatamodelTests(WebApplicationFactory<DatamodelsController> factory) : base(factory)
     {
-        TargetTestRepository = Guid.NewGuid().ToString();
+        TargetTestRepository = GenerateTestRepoName();
```

Comment:
Maybe use `TestDataHelper` directly instead of using static directive for sake of consistency